package-manager/pkg/packmgr/installer_test.go

```
@@ -223,18 +260,26 @@ func TestInstallRejectsInvalidSource(t *testing.T) {
 	}{
 		{name: "missing manifest", mutate: func(_ string) {}},
 		{name: "invalid manifest json", mutate: func(dir string) {
-			os.WriteFile(filepath.Join(dir, "manifest.json"), []byte("{"), 0o640)
+			if err := os.WriteFile(filepath.Join(dir, "manifest.json"), []byte("{"), 0o640); err != nil {
+				t.Fatal(err)
+			}
 		}},
 		{name: "missing artifact", mutate: func(dir string) {
 			writeSourcePackage(t, dir, "demo.pkg", "1.0.0", packagecontract.ModeHosted, "app.wasm", nil)
 			os.Remove(filepath.Join(dir, "app.wasm"))
```

golangci-lint (2.13.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)